---
title: 'Variant'
icon: 'virus'
---

# Variant

The **IGVF Catalog** contains *variant* nodes primarily sourced from [FAVOR](https://favor.genohub.org/).  
Variants not present in FAVOR but present in the other variant collections below were also loaded.  
Variants may be identified using **SPDI**, **HGVS**, or their **genomic position**.

---

## Variant Edges

| Source                | Edge Description                                                                                       | Datasets                                                                                                                                                                                                 |
|---------------------------|---------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| **TopLD**                 | Variant-to-variant edges describing variants in **linkage disequilibrium (LD)**.                  | [TopLD](http://topld.genetics.unc.edu/)                                                                                                                                                                            |
| **caQTL**                   | Variant-to-genomic element edges describing **variants associated with chromatin accessibility**. | [ENCODE](http://encodeproject.org/ENCFF103XRK), [AFGR](https://github.com/smontgomlab/AFGR)                                                                                                                         |
| **eQTL**                    | Variant-to-gene edges describing **variants associated with gene expression**.                   | [EBI eQTL](https://www.ebi.ac.uk/eqtl/), [AFGR](https://github.com/smontgomlab/AFGR)                                                                                                                                |
| **sQTL**                    | Variant-to-gene edges describing **variants associated with alternative splicing**.               | [AFGR](https://github.com/smontgomlab/AFGR)                                                                                                                                                                        |
| **Variant-EFFECTS**         | Variant-to-gene edges describing **causal effects of variants on endogenous gene expression** from genome editing experiments. | [Variant-EFFECTS datasets (Jesse Engreitz, Stanford)](https://data.igvf.org/search/?type=AnalysisSet&status=released&preferred_assay_titles=Variant-EFFECTS&samples.classifications=cell+line&lab.title=Jesse+Engreitz%2C+Stanford) |
| **STARR-seq**                | Variant-to-biosample edges describing **allele-specific regulatory activity** in the K562 cell line. | [STARR-seq datasets (Tim Reddy, Duke)](https://data.igvf.org/search/?type=AnalysisSet&status=released&preferred_assay_titles=STARR-seq&lab.title=Tim+Reddy%2C+Duke&collections=IGVF_catalog_v1.0)                   |
| **BlueSTARR**              | Variant-to-genomic element edges describing **predicted regulatory activity** in ENCODE cCRE sequences using a STARR-seq–trained model. | [BlueSTARR dataset (Bill Majoros, Duke)](https://data.igvf.org/prediction-sets/IGVFDS2340WJRV/)                   |
| **MPRA**                   | Variant-to-genomic element edges describing **variant activity within genomic elements** tested in MPRA. | [lentiMPRA datasets (Nadav Ahituv, UCSF)](https://data.igvf.org/search/?type=AnalysisSet&status=released&preferred_assay_titles=lentiMPRA&collections=IGVF_catalog_v1.0)                                                                 |## Enhancer-Gene Model Prediction Table

This table shows which genes are predicted to be regulated by enhancers overlapping the variant you're viewing.

| Column | Description |
|--------|-------------|
| Cell Type | The specific cell type for the prediction |
| Target Gene | The gene predicted to be regulated (click to view more details) |
| Score | The strength of the prediction (higher is stronger) |
| Dataset | The source dataset (click for more information) |
| Model | The predictive model used |
| Variant-Gene Distance | Physical distance between the variant and gene |

The table is initially sorted by Score in descending order, showing the strongest predictions first.

## Variants in Linkage Disequilibrium Table

This table displays information about variants that are in linkage disequilibrium (LD) with the current variant.

| Column | Description |
|--------|-------------|
| rsID | The reference SNP ID (click for more details) |
| LD (r²) and LD (D') | Measures of linkage disequilibrium |
| Cell Types w/ pred. EG link | Cell types with predicted enhancer-gene links |
| Genes w/ pred. EG link | Genes with predicted enhancer-gene links |
| Ancestry | The population ancestry for the LD information |
| Most Severe Consequence | The predicted functional impact |

You can filter this table by ancestry using the dropdown menu above the table.

## GWAS Association Table

This table shows phenotypes (traits or diseases) associated with the variant through genome-wide association studies (GWAS).

| Column | Description |
|--------|-------------|
| Phenotype Term | The associated trait or disease |
| Study Details | Information about the study (ID, name, author, etc.) |
| p-value | The statistical significance of the association |
| Beta | The effect size and direction |

## Coding Variant Prediction Table

If the variant affects a protein-coding region, this table provides predictions about its impact on protein function.

| Column | Description |
|--------|-------------|
| Gene and Transcript Information | Details about the affected gene and transcript |
| Protein Change | How the variant changes the protein sequence |
| Prediction Scores | Various tools' predictions of the variant's impact (higher scores generally indicate more severe impacts) |

Each table can be sorted by clicking on the column headers. You can also use the search box above each table to filter the results.


